hyper/http20/connection.py

```diff
@@ -507,7 +507,6 @@ def _new_stream(self, stream_id=None, local_closed=False):
         """
         Returns a new stream object for this connection.
         """
-        window_size = self._settings[SettingsFrame.INITIAL_WINDOW_SIZE]
```

Comment:
I think this being missing is a bug: looks like it should be the argument to `__wm_class()`